src/faldbt/project.py

```diff
@@ -634,32 +640,40 @@ def _firestore_dict_to_document(data: Dict, key_column: str):
     return output
 
 
-def _map_nodes_to_models(
+def _map_nodes(
     run_results: DbtRunResult, manifest: DbtManifest, generated_models: Dict[str, Path]
 ):
     models = manifest.get_models()
     tests = manifest.get_tests()
+    sources = [
+        DbtSource(
+            name=source.source_name, table_name=source.name, unique_id=source.unique_id
+        )
+        for source in manifest.get_sources()
+    ]
```

Comment:
What about receiving a source object directly instead of only the parts we need? 
```suggestion
    sources = [DbtSource(source) for source in manifest.get_sources()]
```

Reply:
We will do a separate refactor where we will re-evaluate use of dataclasses in this file